api/api_test.go

```diff
@@ -33,6 +33,18 @@ var issuesRemovableProtected = hstspreload.Issues{
 	},
 }
 
+var issuesRemovableSubdomain = hstspreload.Issues{
+	Errors: []hstspreload.Issue{
+		{Code: "server.removable.subdomain"},
+	},
+}
+
+var issuesRemovablePreloadedTld = hstspreload.Issues{
+	Errors: []hstspreload.Issue{
+		{Code: "server.removable.preloaded_tld"},
```

Comment:
Nit: we use `server.remove.*` already.
```suggestion
		{Code: "server.remove.preloaded_tld"},
```